helix-term/src/ui/editor.rs

```diff
@@ -1180,19 +1181,21 @@ impl EditorView {
             }
 
             MouseEventKind::Drag(MouseButton::Left) => {
-                let (view, doc) = current!(cxt.editor);
+                let editor = &mut cxt.editor;
 
-                let pos = match view.pos_at_screen_coords(doc, row, column) {
-                    Some(pos) => pos,
-                    None => return EventResult::Ignored(None),
-                };
+                if let Some((pos, view_id)) = pos_and_view(editor, row, column) {
```

Comment:
Why the change from `pos_at_screen_coords` on the current view to `pos_and_view` here? Dragging should only affect the currently focused view: with this change, if you start a drag on a focused view and then drag across an unfocused view, you modify the selection on the unfocused view but you can't see the selection because selections are hidden on unfocused views.

Reply:
I did not realize that there are different between `pos_at_screen_coords`  and  `pos_and_view` beside beside `pos_and_view` also return view.id. I will change it.